third_party/proto/cosmos/auth/v1beta1/query.proto

```diff
@@ -7,45 +7,55 @@ import "google/protobuf/any.proto";
 import "google/api/annotations.proto";
 import "cosmos/auth/v1beta1/auth.proto";
 import "cosmos_proto/cosmos.proto";
+import "cosmos/query/v1/query.proto";
 
 option go_package = "github.com/cosmos/cosmos-sdk/x/auth/types";
 
 // Query defines the gRPC querier service.
 service Query {
-  // Accounts returns all the existing accounts
+  // Accounts returns all the existing accounts.
+  //
+  // When called from another module, this query might consume a high amount of
+  // gas if the pagination field is incorrectly set.
   //
   // Since: cosmos-sdk 0.43
   rpc Accounts(QueryAccountsRequest) returns (QueryAccountsResponse) {
-    option (google.api.http).get = "/cosmos/auth/v1beta1/accounts";
+    option (cosmos.query.v1.module_query_safe) = true;
```

Comment:
Note to self: investigate what this `cosmos.query.v1.module_query_safe` means/does.